### PR TITLE
Make each build fail as soon as a job fails

### PR DESCRIPTION
**Pull request by nickmccurdy**

### CODE REVIEW

**Comment by nickmccurdy** [comment]
https://app.clubhouse.io/mycrypto/story/690/make-each-build-fail-as-soon-as-a-job-fails

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ matrix:
       dist: trusty
       sudo: required
       services: docker
+  fast_finish: true
```
Comment (vilsbole) [inline .travis.yml]: fantastically simple!
Is it normal that the indenting is different?

Comment (nickmccurdy) [reply .travis.yml]: It's indented only one level (two spaces) below `matrix:` because it's a setting on matrix, not a specific OS in `include:`

**Comment by nickmccurdy** [comment]
Ready to merge

**Pull request merged**